install_pim/docker/installation_docker.rst

```diff
@@ -38,9 +38,28 @@ Otherwise ``docker-compose`` will create it, but only with root accesses. Then f
 Run and stop the containers
 ---------------------------
 
-**All "docker-compose" commands are to be ran from the folder containing the compose file.**
+.. note::
+
+   All "docker-compose" commands are to be ran from the folder containing the compose file.
+
+.. warning::
+
+   To run the Elasticsearch container, you will probably need to `increase the MAX_MAP_COUNT Linux kernel setting <https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html#docker-cli-run-prod-mode>`_.
```

Comment:
'probably' sounds a bit vague for documentation... you could say 'might need to'